stdlib/xml/dom/minicompat.pyi

```diff
@@ -1,14 +1,14 @@
-from collections.abc import Iterable
-from typing import Any, TypeVar
+from collections.abc import Iterable, Sequence
+from typing import Any, Protocol, TypeVar
 from typing_extensions import Literal
 
 __all__ = ["NodeList", "EmptyNodeList", "StringTypes", "defproperty"]
 
-_T = TypeVar("_T")
+_T = TypeVar("_T", covariant=True)
 
 StringTypes: tuple[type[str]]
 
-class NodeList(list[_T]):
+class NodeList(Sequence[_T], Protocol):
```

Comment:
This doesn't look right. At runtime `NodeList` inherits from `list` and it is indeed a list and its append method is used elsewhere in the code. Why was it changed?